docs/topics/data-analysis/data-analysis-work-with-data-sources.md

```diff
@@ -1,22 +1,179 @@
 [//]: # (title: Retrieve data from files)
 
-Simple guide how to start working data sources with Kotlin notebooks.
+Kotlin Notebook, coupled with the  [DataFrame library](https://kotlin.github.io/dataframe/gettingstarted.html), enables 
```

Comment:
```suggestion
Kotlin Notebook, coupled with the  [Kotlin DataFrame library](https://kotlin.github.io/dataframe/gettingstarted.html), enables 
```

Reply:
Just like for the other docs - I updated it for consistency :) 👍